platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/model/internal/deterministic/DeterministicTaskScheduler.java

```diff
@@ -80,7 +80,7 @@ public long getUnprocessedTaskCount() {
      */
     @Override
     public void flush() {
-        throw new UnsupportedOperationException("Flush is not supported on deterministic task schedulers");
+        // Future work: flushing is incompatible with deterministic task schedulers
```

Comment:
can you just shortly explain why

Reply:
Updated comment:

```
        // Future work: flushing is incompatible with deterministic task schedulers.
        // This is because flushing currently requires us to block thread A while 
        // thread B does work, but with turtle there is only a single thread.
```

Reply:
Something like this, I don't have the solution 100% solved yet. It's a problem I intend to work on in the near future.